{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"multipleErrorsOccurred"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"])"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/coregpx\/gpxerror\/parser\/multipleerrorsoccurred(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError\/parser\/multipleErrorsOccurred(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"When multiple errors occurred, to give an array of errors."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"multipleErrorsOccurred"},{"kind":"text","text":"(["},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"])"}],"title":"GPXError.parser.multipleErrorsOccurred(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:7CoreGPX8GPXErrorV6parserO22multipleErrorsOccurredyAESays5Error_pGcAEmF","modules":[{"name":"CoreGPX"}]},"hierarchy":{"paths":[["doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX","doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError","doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError\/parser"]]},"references":{"doc://org.cocoapods.CoreGPX/documentation/CoreGPX/GPXError/parser":{"role":"symbol","title":"GPXError.parser","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"parser"}],"abstract":[{"type":"text","text":"Parser related errors"}],"identifier":"doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError\/parser","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"parser"}],"url":"\/documentation\/coregpx\/gpxerror\/parser"},"doc://org.cocoapods.CoreGPX/documentation/CoreGPX":{"role":"collection","title":"CoreGPX","abstract":[],"identifier":"doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX","kind":"symbol","type":"topic","url":"\/documentation\/coregpx"},"doc://org.cocoapods.CoreGPX/documentation/CoreGPX/GPXError/parser/multipleErrorsOccurred(_:)":{"role":"symbol","title":"GPXError.parser.multipleErrorsOccurred(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"multipleErrorsOccurred"},{"kind":"text","text":"(["},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"])"}],"abstract":[{"type":"text","text":"When multiple errors occurred, to give an array of errors."}],"identifier":"doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError\/parser\/multipleErrorsOccurred(_:)","kind":"symbol","type":"topic","url":"\/documentation\/coregpx\/gpxerror\/parser\/multipleerrorsoccurred(_:)"},"doc://org.cocoapods.CoreGPX/documentation/CoreGPX/GPXError":{"role":"symbol","title":"GPXError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"GPXError"}],"abstract":[{"type":"text","text":"Throwable errors for GPX library"}],"identifier":"doc:\/\/org.cocoapods.CoreGPX\/documentation\/CoreGPX\/GPXError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"GPXError"}],"url":"\/documentation\/coregpx\/gpxerror"}}}